docs/docs/multiplayer/time.md

```diff
@@ -0,0 +1,82 @@
+---
+sidebar_position: 61
+---
+
+# Time
+
+## Game time
+
+If your game requires access to time, Rune exposes `Rune.gameTimeInSeconds()` that you can use inside logic.js.
+Calling it returns the amount of seconds that have passed since the start of the game.
+
+At the moment Rune allows timers with an accuracy of a second.
+
+For instance this could be used to track how long it took for user to make a guess:
+
+```javascript
+// logic.js
+
+Rune.initLogic({
+  setup() {
+    return {
+      guessRequestedAt: 0,
+      guessDuration: 0
+    };
+  },
+  actions: {
+    requestGuess(_, {game}) {
+      game.guessRequestedAt = Rune.gameTimeInSeconds();
+    },
+    confirmGuess(_, {game}) {
+      game.guessDuration = Rune.gameTimeInSeconds() - game.guessDuration;
+    }
+  }
+})
+```
```

Comment:
I think it might also be good to use examples that support multiple simultaneous players. We could consider using the examples from the initial time-syncing scope document?

Reply:
I've switched to quiz example. I just have a feeling that it is a bit too complicated to easily indicate the usage of gameTimeInSeconds.